resource_customizations/kyverno.io/Policy/testdata/healthy.yaml

```diff
@@ -0,0 +1,261 @@
+        cloneList: {}
+      match:
+        resources:
+          kinds:
+          - CronJob
+      mutate: {}
+      name: autogen-cronjob-require-annotation-version
+      skipBackgroundRequests: true
+      validate:
+        message: A `version` annotation is required.
+        pattern:
+          spec:
+            jobTemplate:
+              spec:
+                template:
+                  metadata:
+                    annotations:
+                      version: ?*
+    - exclude:
+        resources: {}
+      generate:
+        clone: {}
+        cloneList: {}
+      match:
+        resources:
+          kinds:
+          - DaemonSet
+          - Deployment
+          - Job
+          - StatefulSet
+          - ReplicaSet
+          - ReplicationController
+      mutate: {}
+      name: autogen-require-annotation-maintainer
+      skipBackgroundRequests: true
+      validate:
+        message: A `maintainer` annotation is required.
+        pattern:
+          spec:
+            template:
+              metadata:
+                annotations:
+                  maintainer: ?*
+    - exclude:
+        resources: {}
+      generate:
+        clone: {}
+        cloneList: {}
+      match:
+        resources:
+          kinds:
+          - CronJob
+      mutate: {}
+      name: autogen-cronjob-require-annotation-maintainer
+      skipBackgroundRequests: true
+      validate:
+        message: A `maintainer` annotation is required.
+        pattern:
+          spec:
+            jobTemplate:
+              spec:
+                template:
+                  metadata:
+                    annotations:
+                      maintainer: ?*
+    - exclude:
+        resources: {}
+      generate:
+        clone: {}
+        cloneList: {}
+      match:
+        resources:
+          kinds:
+          - DaemonSet
+          - Deployment
+          - Job
+          - StatefulSet
+          - ReplicaSet
+          - ReplicationController
+      mutate: {}
+      name: autogen-require-spec-containers
+      skipBackgroundRequests: true
+      validate:
+        message: The Pod must have at least one container.
+        pattern:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: ?*
+    - exclude:
+        resources: {}
+      generate:
+        clone: {}
+        cloneList: {}
+      match:
+        resources:
+          kinds:
+          - CronJob
+      mutate: {}
+      name: autogen-cronjob-require-spec-containers
+      skipBackgroundRequests: true
+      validate:
+        message: The Pod must have at least one container.
+        pattern:
+          spec:
+            jobTemplate:
+              spec:
+                template:
+                  spec:
+                    containers:
+                    - name: ?*
+  conditions:
+  - lastTransitionTime: "2025-01-17T19:09:11Z"
+    message: Ready
+    reason: Succeeded
+    status: "True"
+    type: Ready
+  ready: true
```

Comment:
```suggestion
```

`status.ready` field got deprecated in this [PR](https://github.com/kyverno/kyverno/pull/10999). Was using an older kyverno version locally in my development environment.